src/pal/src/thread/thread.cpp

```diff
@@ -21,6 +21,8 @@ Module Name:
 #include "pal/dbgmsg.h"
 SET_DEFAULT_DEBUG_CHANNEL(THREAD); // some headers have code with asserts, so do this first
 
+#include "stdmacros.h"
```

Comment:
I agree. And if we need something from the stdmacros.h, I would rather redefine it for PAL usage instead.
The change in the stdmacros.h should also be reverted then.